cmd/cloud-server/logics/load-balancer/create_url_rule_validator.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * TencentBlueKing is pleased to support the open source community by making
+ * 蓝鲸智云 - 混合云管理平台 (BlueKing - Hybrid Cloud Management System) available.
+ * Copyright (C) 2022 THL A29 Limited,
+ * a Tencent company. All rights reserved.
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://opensource.org/licenses/MIT
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * We undertake not to change the open source license (MIT license) applicable
+ *
+ * to the current version of the project delivered to anyone in the future.
+ */
+
+package lblogic
+
+import (
+	"encoding/json"
+
+	dataservice "hcm/pkg/client/data-service"
+	"hcm/pkg/criteria/enumor"
+	"hcm/pkg/kit"
+	"hcm/pkg/logs"
+)
+
+var _ ImportValidator = (*CreateUrlRuleValidator)(nil)
+
+func newCreateUrlRuleValidator(cli *dataservice.Client, vendor enumor.Vendor, bkBizID int64,
+	accountID string, regionIDs []string) *CreateUrlRuleValidator {
+
+	return &CreateUrlRuleValidator{
+		newBasePreviewExecutor(cli, vendor, bkBizID, accountID, regionIDs),
+	}
+}
+
+// CreateUrlRuleValidator ...
+type CreateUrlRuleValidator struct {
+	*basePreviewExecutor
+}
+
+// Validate ...
+func (c *CreateUrlRuleValidator) Validate(kt *kit.Kit, rawData json.RawMessage) (interface{}, error) {
+	validator := &CreateUrlRulePreviewExecutor{
+		basePreviewExecutor: c.basePreviewExecutor,
+	}
```

Comment:
全部改成, validator容易和validator包混淆
```suggestion
	executor := &CreateUrlRulePreviewExecutor{
		basePreviewExecutor: c.basePreviewExecutor,
	}
```